stream/classes/cost_model/cost_model.py

```diff
@@ -14,10 +15,10 @@ class StreamCostModelEvaluation:
 
     def __init__(
         self,
-        workload: DiGraph,
+        workload: Workload,
         accelerator: Accelerator,
-        operands_to_prefetch: list,
-        scheduling_order: list,
+        operands_to_prefetch: list[str],
```

Comment:
Not sure, but should this not be `MemoryOperand`?

Reply:
Good catch, but no. Almost all user input is now provided through the yaml files, and memory operand names are parsed as `MemoryOperand`. The `operands_to_prefetch` is provided in the `main.py` file itself. I didn't want to let the user convert it himself in the definition of  `operands_to_prefetch`, so it is converted to the correct class within the function that uses ``operands_to_prefetch`.